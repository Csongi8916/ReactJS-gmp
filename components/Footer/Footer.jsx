import React from 'react'
import Logo from '../Logo/Logo';
import './Footer.scss';

export default function Footer() {
  return (
    <div className="Footer">
      <Logo />
    </div>
  )
}
