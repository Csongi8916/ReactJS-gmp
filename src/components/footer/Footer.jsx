import React from 'react'
import Logo from '../Logo/Logo';
import classes from './Footer.scss';

export default function Footer() {
  return (
    <div className="Footer">
      <Logo />
    </div>
  )
}
