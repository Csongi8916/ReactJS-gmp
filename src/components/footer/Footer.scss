$nrBgColor: #555555;

.Footer {
  display: flex;
  justify-content: center;
  align-items: center;
  min-height: 60px;
  background-color: $nrBgColor;
}