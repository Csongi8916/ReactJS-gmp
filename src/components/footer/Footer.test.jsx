import React from 'react';
import renderer from 'react-test-renderer';
import Footer from './Footer';

test('two plus two is four', () => {
  expect(2 + 2).toBe(4);
});

describe('Footer', () => {
  test('renders correctly', () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
