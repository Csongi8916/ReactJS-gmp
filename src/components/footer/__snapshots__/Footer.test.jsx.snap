// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<div
  className="Footer"
>
  <span
    className="root"
  >
    <span
      className="main-part"
    >
      netflix
    </span>
    <span>
      rolulette
    </span>
  </span>
</div>
`;
